src/components/Tooltip/TooltipContentWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+interface TooltipContentWrapperProps {
+  children: React.ReactNode // Accepts any JSX
+  className?: string // Allow styling if needed
+}
+
+export function TooltipContentWrapper({
```

Comment:
this wrapper is only for styling? Can you not just put a div around content in the tooltip wrapper?